presto-main/src/main/java/io/prestosql/server/security/UserMapping.java

```diff
@@ -84,33 +86,57 @@ public UserMappingRules(
         }
     }
 
+    enum Case
+    {
```

Comment:
We also need to update documentation

Reply:
where is it?

Reply:
https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/security/user-mapping.rst